medusa/helpers.py

```diff
@@ -716,7 +716,7 @@ def sanitize_scene_name(name, anime=False):
         name = name.replace(x, '')
 
     # tidy up stuff that doesn't belong in scene names
-    name = name.replace('- ', '.').replace(' ', '.').replace('&', 'and').replace('/', '.')
+    name = name.replace('- ', '.').replace(' ', '.').replace('&', 'and').replace('/', '.').replace(': ', '-')
```

Comment:
I don't know if all sites' search engines return results with a space in a character's place, on the other hand with the dash i do know they will, becouse thats how the releases are named.